core/jobs_registry.py

```diff
@@ -48,6 +48,7 @@
     story_jobs_one_off.StoryMigrationOneOffJob,
     suggestion_jobs_one_off.QuestionSuggestionMigrationJobManager,
     topic_jobs_one_off.TopicMigrationOneOffJob,
+    topic_jobs_one_off.UpdateTopicThumbnailSizeOneOffJob
```

Comment:
Perhaps call it PopulateTopicThumbnailSizeOneOffJob

In the job, don't do anything if the thumbnail size in bytes is already not-None. That way, the job is idempotent (which is an important property for these jobs). I.e. if it fails midway in its first run, then running it again would give the same result as if it had run fully correctly the first time.

Reply:
Done.